src/ess/loki/data.py

```diff
@@ -31,9 +37,48 @@
 )
 
 
-def get_path(filename: FilenameType) -> FilePath[FilenameType]:
-    """Translate any filename to a path to the file obtained from pooch registries."""
-    return FilePath[FilenameType](_registry.get_path(filename))
+def loki_tutorial_sample_run_60250() -> Filename[SampleRun]:
+    """Sample run with sample and sample holder/can, no transmission monitor in beam."""
+    return Filename[SampleRun](_registry.get_path('60250-2022-02-28_2215.nxs'))
+
+
+def loki_tutorial_sample_run_60339() -> Filename[SampleRun]:
+    """Sample run with sample and sample holder/can, no transmission monitor in beam."""
+    return Filename[SampleRun](_registry.get_path('60339-2022-02-28_2215.nxs'))
+
+
+def loki_tutorial_background_run_60248() -> Filename[BackgroundRun]:
+    """Background run with sample holder/can only, no transmission monitor."""
+    return Filename[BackgroundRun](_registry.get_path('60248-2022-02-28_2215.nxs'))
+
+
+def loki_tutorial_background_run_60393() -> Filename[BackgroundRun]:
+    """Background run with sample holder/can only, no transmission monitor."""
+    return Filename[BackgroundRun](_registry.get_path('60393-2022-02-28_2215.nxs'))
+
+
+def loki_tutorial_sample_transmission_run() -> Filename[TransmissionRun[SampleRun]]:
+    """Sample transmission run (sample + sample holder/can + transmission monitor)."""
+    return Filename[TransmissionRun[SampleRun]](
+        _registry.get_path('60394-2022-02-28_2215.nxs')
+    )
+
+
+def loki_tutorial_run_60392() -> Filename[TransmissionRun[BackgroundRun]]:
```

Comment:
```suggestion
def loki_tutorial_background_transmission_run() -> Filename[TransmissionRun[BackgroundRun]]:
```

Reply:
Maybe we can just also have an `loki_tutorial_empty_beam_run()` function that returns the same file name, which could then be changed to return something else in the future if we need that?

Reply:
Then the user-visible code is "lying", since it will be invisible that the same run is used. I can change it if you prefer?

Reply:
It's fine, you can leave it as is.